pisa/utils/hyperplane.py

```diff
@@ -0,0 +1,628 @@
+import numpy as np
```

Comment:
always always use
```from __future__ import absolute_import, division, print_function```
at the top of your imports; see comment below about py3 compatibility